### ISSUE TITLE
CUDA DISABLED (THE SPECIFIED MODULE COULD NOT BE FOUND) 

### ISSUE THREAD

**Issue by Stayblonded** [xmrig/xmrig-nvidia#320]
I'm new to crypto and decided to mine monero wit xmrig. It's been going great but I wanted to install the cuda plugin and use my gpu. I put the files into the xmrig folder and gave it the path c:\\users\\lopay\\desktop\\xmrig-6.4.0\\xmrig-cuda.dll in the config. However, it keeps saying that the "specified module could not be found. So I tried changing the path to other cuda plugin files to fix it. It didn't fix it but it started saying "symbol not found: version". Then I tried another file and it said "cudeviceprimaryctxrelease_v2 not found". I have no idea how to fix this. Help pls
![Screenshot (1)](https://user-images.githubusercontent.com/73660178/97606813-73663400-19cd-11eb-8c00-75bc06ecab9b.png)
![Screenshot (2)](https://user-images.githubusercontent.com/73660178/97606816-73feca80-19cd-11eb-813b-a68f8403025e.png)

**Comment by atamsetya** [comment]
have solution for this??

**Comment by riverdusty** [comment]
bump

**Comment by therealkushsmoke** [comment]
Did you download the [CUDA plugin](https://github.com/xmrig/xmrig-cuda/releases)? This needs to be put in the xmrig folder.

**Comment by cuppajoeman** [comment]
@therealkushsmoke That works for me.

Note: when I extract the contents of the plugin to xmrig-X.Y.Z I didn't replace the SHA256SUMS file.

**Comment by cyb3rko** [comment]
@cuppajoeman Thanks, the note that you have to extract the files into the xmrig folder without the ```SHA256SUMS``` file helped me.

**Comment by hackcrow** [comment]
isn't work for me, I  extracted the files into the xmrig folder without the SHA256SUMS, It's show the same error

**Comment by agnel** [comment]
> @therealkushsmoke That works for me.
> 
> Note: when I extract the contents of the plugin to xmrig-X.Y.Z I didn't replace the SHA256SUMS file.

Yes, This worked for me. Earlier it didn't work because I extracted the contents of the cuda plugin to a folder and then I had placed this folder where xmrig-x.y.z.exe resides.

Thanks.

**Comment by DeeDeeRanged** [comment]
xmrig-nvidia is abandoned got to https://github.com/xmrig/xmrig.
In any case you just dump the xmrig-cuda.dll in the same folder where xmrig is. There is no need for a path to it aslong as its all in the same directory.
